modules/nw-ingress-creating-a-passthrough-route.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * ingress/routes.adoc
+
+[id="nw-ingress-creating-a-passthrough-route_{context}"]
+= Creating a passthrough route
+
+You can configure a secure route using passthrough termination by using the `oc create route` command. With passthrough termination, encrypted traffic is sent straight to the destination without the router providing TLS termination. Therefore no key or certificate is required.
+
+.Prerequisites
+
+* You must have a service that you want to expose.
+
+.Procedure
+
+* Create a `Route` resource:
++
+[source,terminal]
+----
+$ oc create route passthrough route-passthrough-secured --service=frontend --port=8080
+----
++
+If you examine the resulting `Route` resource, it should look similar to the following:
++
+.A Secured Route Using Passthrough Termination
+[source,yaml]
+----
+apiVersion: v1
+kind: Route
+metadata:
+  name: route-passthrough-secured <1>
+spec:
+  host: www.example.com
+  port:
+    targetPort: 8080
+  tls:
+    termination: passthrough <2>
+  to:
+    kind: Service
+    name: frontend
+----
+<1> The name of the object, which is limited to 63 characters.
+<2> The `*termination*` field is set to `passthrough`. No other encryption fields are needed.
++
+The destination pod is responsible for serving certificates for the
+traffic at the endpoint. This is currently the only method that can support
+requiring client certificates (also known as two-way authentication).
```

Comment:
Sorry I didn't catch this before! The IBM style guide advises against using parenthesis.  

Suggested edits: 

This is currently the only method that can support
requiring client certificates, also known as two-way authentication.

The only valid values are `None`, empty for disabled, or `Redirect`.